blockchain/chaingen/generator.go

```diff
@@ -337,8 +337,10 @@ func (g *Generator) calcFullSubsidy(blockHeight uint32) dcrutil.Amount {
 // good tests which exercise that code, so it wouldn't make sense to use the
 // same code to generate them.
 func (g *Generator) calcPoWSubsidy(fullSubsidy dcrutil.Amount, blockHeight uint32, numVotes uint16) dcrutil.Amount {
-	powProportion := dcrutil.Amount(g.params.WorkRewardProportion)
-	totalProportions := dcrutil.Amount(g.params.TotalSubsidyProportions())
+	const (
```

Comment:
Was expecting to see the new reward split here. I'm guessing you're opting for updating the constants once the agenda is active?

Reply:
It's the same logic as it currently is where it only uses the legacy values in chaingen.  Any blocks that are generated with the new rules use mungers to update to the new values.

Reply:
Noted, thanks.